test/extended/util/framework.go

```diff
@@ -160,3 +268,14 @@ func ExtendedTestPath() string {
 func FixturePath(elem ...string) string {
 	return filepath.Join(append([]string{ExtendedTestPath()}, elem...)...)
 }
+
+// MustParseLabelsOrDie turns the given string itno a label selector or
+// panics; for tests or other cases where you know the string is valid.
+// TODO: Move this to the upstream labels package.
+func MustParseLabelsOrDie(str string) labels.Selector {
```

Comment:
ParseLabelsOrDie ... there is no need for Must (and when we talked trough this I told you... shorter is better).